modules/k8s-nmstate-troubleshooting-dns-disconnected-env.adoc

```diff
@@ -15,8 +15,7 @@ Ensure that the DNS server includes a name server (NS) entry for the `root-serve
 
 == Configuring the bind9 DNS named server  
```

Comment:
Out-of-scope, but addressing a concern: Procedures can only have one heading. Please keep in mind, once we migrate to DITA, block titles (ie `.Example`) won't work for items besides examples, tables, and features. https://docs.google.com/presentation/d/1yWZfJybc4SVxgXg4ZCmuBleuFpf8jSMEQkRar39mfdw/edit#slide=id.g304c4c806dd_0_53

Reply:
Thanks. I'll leave the structure as is for now until the DITA migration path is clearer.